### PR TITLE
Gqm model

### PR DESCRIPTION
**Pull request by drunking**

### CODE REVIEW

**Comment by eliseev-eag** [comment]
Конфликты реши

**Comment by drunking** [comment]
> Конфликты реши

drunking@Alexanders-MBP:~/dev/SberBoard$ git push origin gqm-model
Enumerating objects: 16, done.
Counting objects: 100% (16/16), done.
Delta compression using up to 8 threads
Compressing objects: 100% (5/5), done.
Writing objects: 100% (6/6), 585 bytes | 585.00 KiB/s, done.
Total 6 (delta 2), reused 0 (delta 0)
remote: Resolving deltas: 100% (2/2), completed with 2 local objects.
remote: error: GH006: Protected branch update failed for refs/heads/gqm-model.
remote: error: At least 1 approving review is required by reviewers with write access.
To github.com:eliseev-eag/SberBoard.git
 ! [remote rejected] gqm-model -> gqm-model (protected branch hook declined)
error: failed to push some refs to 'git@github.com:eliseev-eag/SberBoard.git'

File: src/main/java/ru/sb/sboard/_common/domain/AbstractIdentity.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package ru.sb.sboard._common.domain;
```
[the remark this answers is not in the document]
Comment (drunking) [reply src/main/java/ru/sb/sboard/_common/domain/AbstractIdentity.java]: Можно и без него, с ним пакет будет вверху списка, просто для удобства.

[the remark this answers is not in the document]
Comment (drunking) [reply src/main/java/ru/sb/sboard/_common/domain/AbstractIdentity.java]: Убрал

File: src/main/java/ru/sb/sboard/gqm/controller/GqmController.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+@RestController
+@RequestMapping("/gqm/")
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class GqmController {
+
+    private final GoalRepository goalRepository;
+    private final QuestionRepository questionRepository;
+
+    @PostMapping("goal")
```
[the remark this answers is not in the document]
Comment (drunking) [reply src/main/java/ru/sb/sboard/gqm/controller/GqmController.java]: Убрал избыточный контроллер.

**Pull request merged**